### PR TITLE
File Hierarchy clean up

### PR DESCRIPTION
**Pull request by rushirajnenuji**
Move bytestoSize to Utilities Ref #2484
[WIP] FH ES Lint issues #2483

### CODE REVIEW

**Review by robyngit — APPROVED** [review]
It's good to have the `bytesToSize` consolidated into one location! Everything seems to still work & I see that your ES-lint changes are a WIP so we can ignore the remaining errors for this PR.

Approve conditional on a run of `npm run format` (`DataPackage` has some prettier errors). Feel free to merge after that! 🎉

**Comment by robyngit** [comment]
@rushirajnenuji, I pushed a little fix for the formatting issue with `DataPackage` hoping to get the branch merged. There remain two small conflicts with develop in the `MetadataView` that need to be resolved before it can be merged.

**Comment by rushirajnenuji** [comment]
Thank you for adding the reviewing and testing this PR, and for adding the formatting fix @robyngit , I've got this branch up to speed with `develop`, and is ready for the merge.

**Pull request merged**
